api/job_errors.md

```diff
@@ -0,0 +1,79 @@
+---
+api_name: timescaledb_information.job_errors
+excerpt: Get information about background job errors
+keywords: [jobs, information]
+tags: [background jobs, scheduled jobs, automation framework, scheduled views]
+api:
+  license: community
+  type: view
+---
+
+## timescaledb_information.job_errors
+
+Shows information about runtime errors encountered by jobs run by the automation framework.
+This includes jobs set up for user defined actions and jobs run by policies
+created to manage data retention, continuous aggregates, compression, and
+other automation policies.  (See [policies][actions]).
+
+### Available columns
+
+|Name|Type|Description|
+|---|---|---|
+|`job_id` | INTEGER | The id of the background job created to implement the policy |
+|`proc_schema`| TEXT | Schema name of the function or procedure executed by the job |
+|`proc_name`| TEXT |Name of the function or procedure executed by the job|
+|`pid`| INTEGER | The process id of the background worker executing the job. This is `NULL` in the case of a job crash |
+|`start_time`| TIMESTAMP WITH TIME ZONE | Start time of the job |
+|`finish_time`| TIMESTAMP WITH TIME ZONE | Time when error was reported |
+|`sqlerrcode`| TEXT | The error code associated with this error, if any. See the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/errcodes-appendix.html) for a full list of error codes |
+|`err_message`| TEXT | The detailed error message |
```

Comment:
Table needs to be compressed

Reply:
thank you for reviewing @Loquacity! Not sure what this comment means, what should I change above?

Reply:
```suggestion
|`err_message`|TEXT|The detailed error message|
```

Just take the spaces out, like this ☝️